container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_api_container_functions.go

```diff
@@ -445,7 +444,9 @@ func (backend *DockerKurtosisBackend) getMatchingApiContainers(ctx context.Conte
 			apiContainer.GetLabels(),
 			apiContainer.GetStatus(),
 			apiContainer.GetHostPortBindings(),
-			bridgeNetworkIpAddress,
+			// Bridge network IP address will be filled in later if this APIContainer passes the post-filter.
```

Comment:
should this param even exist anymore then, if we're expecting that that info will get filled in later?